dwa_local_planner/src/dwa_planner_ros.cpp

```diff
@@ -344,8 +360,11 @@ namespace dwa_local_planner {
   }
 
   bool DWAPlannerROS::cancel() {
-  canceled_ = true;
-  return true;
+    // if we cancel, reset the latching
+    ROS_DEBUG_NAMED("dwa_local_planner", "Clearing latching in dwa cancel. How did we get here?");
```

Comment:
are we getting here??

Reply:
no, this is wrapped by the wrapper_local_planner in move base flex, and never called. the log line is just to warn us if that changes in the future. i'll change it to warning.

Reply:
that means the function `resetLatching` will never be called?

Reply:
it's called in several other places in the code, but won't be hit here until we change wrapper_local_planner